tests/ts_simple_batch.erl

```diff
@@ -36,15 +36,41 @@
 -define(UPPER_QRY,  900050).
 
 confirm() ->
-    TestType = normal,
     DDL = ts_util:get_ddl(),
     Qry = ts_util:get_valid_qry(?LOWER_QRY, ?UPPER_QRY),
     Data = ts_util:get_valid_select_data(fun() -> lists:seq(?LOWER_DATA,?UPPER_DATA) end),
     Expected =
         {ts_util:get_cols(small),
          ts_util:exclusive_result_from_data(Data, ?LOWER_QRY-?LOWER_DATA+2, (?LOWER_QRY-?LOWER_DATA)+(?UPPER_QRY-?LOWER_QRY))},
-    Conn = ts_util:cluster_and_connect(single),
-    ts_util:ts_put(Conn, TestType, DDL, Data),
-    Got = ts_util:ts_query(Conn, TestType, DDL, Data, Qry),
+    {[Node], Pid} = ts_util:cluster_and_connect(single),
+
+
+    rt_intercept:add(Node, {riak_kv_eleveldb_backend,
+                            [{{batch_put, 4}, batch_put}]}),
+
+    %% Buried in the bowels of the code path behind ts_util:ts_put/4
+    %% is a calculation that n_val is the same as the cluster size. I
+    %% want a single node cluster for this test, but n_val of 4, so
+    %% I'll duplicate the path here
+    Bucket = ts_util:get_default_bucket(),
+    {ok, _} = ts_util:create_bucket_type([Node], DDL, Bucket, 4),
+    ts_util:activate_bucket_type([Node], Bucket),
```

Comment:
I would prefer `riak_ts:query(rt:pbc(Node), DDL ++ " WITH (nval=4)")`. But it's not something that needs 'fixing' here.